### DISCUSSION TITLE
Add ability to highlight or indicate certain entries in the tree

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5562] [Feature Requests]
Requested by PRI - Leslie
She wants to be able to highlight or in some way indicate one or more entries in the taxon tree.  She said it would make visually searching for items in the tree easier.

"Is there a way or is there any thought to being able to highlight or change font color for individual entries in the Specify Trees?  If we could change certain entries red or highlight them in yellow, it would make visually searching for things easier – especially if you are using a split screen to synonymize something.  We also hve a big update coming for part of our stratigraphic tree.  Being able to highlight something/multiple things in the tree would help make sure that you get all the changes made.  Just a thought. "

**Comment by maxpatiiuk** [comment]
Does she want the highlights to persist between sessions?
Do highlights need to be shared between users?

If the answer to both of these is no, then this feature would take just ~2 hours to implement

**Comment by tlammer** [comment]
For the request, how long would the highlight last for?  Until you clear it or until that session ends?



I don’t know.  If you were just giving someone a task, say to look up/work on certain things, you might just want tit to be for a short time period.  If it was a longer term project you might want it to be for longer.  If you tied it to a query, you could always rerun the query, but that would highlight all the answers again, even if you had fixed some of them unless you had a way or marking that you had done those already.  Sometimes all I want to do is run through a tre using the find next button so I can see everything that might have the name that I am searching for.  This usually happens if it has been entered as a species with a subgenus and if it has been entered as a species in a genus.  Being able to find both easily visually, without having to split the screen would help.
I have not really thought much about how or how long something would last, I was just thinking about how convenient it is to be able to highlight /underline/change font color when you are working in documents.

**Comment by maxpatiiuk** [comment]
The tree redesign includes making the search box display rank names in search results, thus, the need for the "find next" button is reduced.

The split-screen functionality should be doable too. Opened https://github.com/specify/specify7/issues/1148 for that
